index.html

```diff
@@ -97,6 +98,7 @@
     table.simple td {
         border-right: 1px solid #000;
     }
+    .issue[id^='issue-'] > *:not([role='heading']) {}
```

Comment:
I am the one to blame here :-) I introduced @zolkis to the ReSpec-GitHub issue integration (an undocumented feature AFAIK), and proposed the use of CSS to hide the issue description. By default, the issue that gets rendered contains also the first comment from the respective GH issue -- often too elaborate description to be included in the spec.

Now, the issue that made use of this feature seem to have been resolved and removed from the spec thus this CSS is currently not in use.

Anyway, given e.g. the following HTML:

`<div class="issue" data-number="51"></div>`

The rendered end result is something like:

```
<div class="issue" id="issue-51">
<div class="issue-title">
  <a href="https://github.com/w3c/web-nfc/issues/51"><span>Issue 51</span></a>:
  Create per-origin NFC adapter properties section?
</div>
</div>
```

Saves some editorial effort since no manual synching between the spec and GH issues is needed. Also, this way the GH issues can be used as the canonical source for issues (if so preferred). That said, I leave it to the editors to decide how to best manage the issues, i.e. inline or in GH issues or a combination of both.